### PR TITLE
Fix/2539 slider dynamic default value

### PR DESCRIPTION
**Pull request by kwongz**
### Description

Slider accepts Data prop, can use 

1. maxColumn to get the first row value of a column and sets slider's max value
2. minColumn to get the first row value of a column and sets slider's min value
3. defaultValue now accepts strings, gets first row of a column and set slider's default value

### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [x] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

**Comment by kwongz** [comment]
Had some questions on handling the dynamic values

- Would it be for the user to avoid creating new columns (`max_value`, `min_value`)? Instead accept a single `column` (sales) and if `maxColumn`/`minColumn`  prop is provided we provide the max and min values from that column?
- Currently it feels bad to be arbitrarily taking the first value of the max and min columns

**Comment by ItsMeBrianD** [comment]
> Currently it feels bad to be arbitrarily taking the first value of the max and min columns

This is the way `Value` and `BigValue`, so I don't think this is an issue

> Would it be for the user to avoid creating new columns (max_value, min_value)? Instead accept a single column (sales) and if maxColumn/minColumn prop is provided we provide the max and min values from that column?

I'm not sure what the question is asking here

**Comment by kwongz** [comment]
all good, not important

File: packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+			error = e.message;
+			const setTextRed = '\x1b[31m%s\x1b[0m';
+			console.error(setTextRed, `Error in Value: ${error}`);
+			if (strictBuild) {
+				throw error;
+			}
+		}
+	}
+
+	// Query data integration
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte:156]: What's this comment for?

Comment (zachstence) [inline packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte:156]: What's this comment for?

**Review by zachstence — APPROVED** [review]

File: packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte (hunk 1 above)
Comment (kwongz) [reply packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte:156]: that should not be there, will remove

Comment (zachstence) [reply packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte]: [af2d4db](https://github.com/evidence-dev/evidence/pull/2672/commits/af2d4dbf110bf709270453da950befd25bc2f5c2)

**Comment by kwongz** [comment]
fixes #2539

**Pull request merged**